SPECS/xcp-ng-generic-lib.spec

```diff
@@ -0,0 +1,57 @@
+Name:           xcp-ng-generic-lib
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A library of algorithms, I/O and networking functions...
+License:        GPLv3
+URL:            https://github.com/xcp-ng/xcp-ng-generic-lib
+Source0:        https://github.com/xcp-ng/xcp-ng-generic-lib/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
+%description
+A library of algorithms, I/O and networking functions... used by XCP-ng tools or daemons.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
```

Comment:
preferrably one per line (&& is not necessary)

Reply:
Why is it preferable? ^^

Reply:
That's usually the convention in spec files from what I know. Unless really needed, one command per line. And you almost never see `&&` since the build will fail at the first error.